charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -37,22 +37,16 @@ livenessProbe: {}
 readinessProbe: {}
 
 # Set nodeSelector where the controller shut run, i.e. controlplane nodes
```

Comment:
Can you fix the description too? I guess "shut" is "should".

Reply:
Sure, happy to.